app/assets/stylesheets/components/stick-at-top-when-scrolling.scss

```
@@ -1,14 +1,14 @@
 // CSS adapted from
 // https://github.com/alphagov/govuk_frontend_toolkit/blob/d9489a987086471fe30b4b925a81c12cd198c91d/docs/javascript.md#stick-at-top-when-scrolling
 
-$sticky-padding: $gutter * 2 / 3;
+$sticky-padding: govuk-spacing(4);
 
 .js-stick-at-top-when-scrolling,
 .js-stick-at-bottom-when-scrolling {
 
   overflow: hidden;
-  margin-left: -$gutter-half;
-  padding: 10px 0 0 $gutter-half;
+  margin-left: -govuk-spacing(3);
+  padding: 10px 0 0 govuk-spacing(3);
```

`10px` is equivalent to `$gutter-one-third`
```suggestion
  padding: govuk-spacing(2) 0 0 govuk-spacing(3);
```